packages/css-framework/src/objects/_floating-box.scss

```diff
@@ -0,0 +1,231 @@
+$floating-box-light-background: color(neutral, white);
```

Comment:
I wonder if we should prefix these with `_` as they are private and this will be neater going forward. I did this in `_alert.scss`. Maybe it's worth having a discussion to make sure we are happy with the pattern.